### PR TITLE
MGMT-7901: Kube-API import cluster doc

### PR DESCRIPTION
**Pull request by rollandf**
# Assisted Pull Request

## Description
Add documentation on how to import an existing cluster to add additional nodes.

## List all the issues related to this PR

- [ ] New Feature
- [ ] Bug fix
- [ ] Tests
- [x] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [x] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [x] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Assignees

/cc @nmagnezi 
/cc @filanov 
/cc @avishayt 

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by rollandf** [comment]
/hold
Verifying the steps

**Comment by rollandf** [comment]
/hold cancel
Verified manually

File: docs/hive-integration/import-installed-cluster.md
Comment (nmagnezi) [inline docs/hive-integration/import-installed-cluster.md:29]: Why? Perhaps there is a room to elaborate here?

Is that because we do not allow adding day2 hosts to SNO clusters? If yes: maybe we can write it or link to another document that does?
Also: what would happen if the user set this to: `2`? will this pass validations?

**Review by nmagnezi — CHANGES_REQUESTED** [review]
Just one comment, otherwise looks good.

File: docs/hive-integration/import-installed-cluster.md
Comment (rollandf) [reply docs/hive-integration/import-installed-cluster.md:29]: Yes, we do not allow adding day2 to SNO.
 if the user set this to 2, it will this pass validations since Day2 clusters is not looking at this parameter.

Comment (rollandf) [reply docs/hive-integration/import-installed-cluster.md]: Done

**Review by nmagnezi — COMMENTED** [review]
LGTM. I'll let other people to review as well.

File: docs/hive-integration/import-installed-cluster.md
Comment (avishayt) [inline docs/hive-integration/import-installed-cluster.md:37]:
```suggestion
- The `adminKubeconfigSecretRef` is a reference to a secret containing the `kubeconfig`. Here is how to create it (replace `/path/to/kubeconfig` to the location of the kubeconfig file):
```

File: docs/hive-integration/import-installed-cluster.md
Comment (avishayt) [inline docs/hive-integration/import-installed-cluster.md:43]:
```suggestion
- The `adminPasswordSecretRef` is a reference to a secret containing the admin user name and password. Here is how to create it (replace `supersecretname` and `supersecretpassword` to the actual credentials):
```

**Review by avishayt — COMMENTED** [review]
A couple of small nits, otherwise looks good

File: docs/hive-integration/import-installed-cluster.md
Comment (rollandf) [reply docs/hive-integration/import-installed-cluster.md]: Done

File: docs/hive-integration/import-installed-cluster.md
Comment (rollandf) [reply docs/hive-integration/import-installed-cluster.md]: Done

**Review by avishayt — APPROVED** [review]

**Pull request merged**
